mods/ts/rules/defaults.yaml

```diff
@@ -382,6 +382,8 @@
 		ReferencePoint: Bottom, Right
 		RequiresCondition: hospitalheal
 	RevealOnFire:
+	RejectsOrders:
```

Comment:
This is a bad idea for now - if units stop inside tunnels (because Move sucks) then it won't be possible to recover them.